plugins/disk_sizes/plugin.rb

```diff
@@ -10,10 +10,10 @@ def register
                           storage_table: 'disk_size_measurements',
                           units: 'bytes')
     # execute migration
+    path = File.dirname(__FILE__) + '/migrations'
     Sequel::Migrator.run(Iam.settings.DB,
-                         '/data/code/plugins/disk_sizes/migrations',
-                         column: :disk_size_ver
-                        )
+                         path,
```

Comment:
Instead of creating a variable you only use once, can you put `File.dirname(__FILE__) + '/migrations'` here directly?